### PR TITLE
Kelin lab05 pick6

### PR DESCRIPTION
**Pull request by rayk472**
Version 1 pick6 lab. Still working on it.

### CODE REVIEW

File: code/kelin/labs/lab05_pick6.py [hunk 1]
```diff
@@ -1,57 +1,67 @@
...
 
+def pick6(): # First function generates a list of 6 random numbers
+    nums = []
+    while len(nums) < 6:
+        nums.append(random.randint(1, 99))
+    return nums
+
+def num_matches(ticket, winning): # Second function returns the number of matches
+    matches = 0
+    for index in range(0,6):
```
Comment (violette-git) [inline code/kelin/labs/lab05_pick6.py:33]: Our ticket number is a list of 6 random numbers. take a look at the index you are specifying here. 0-6

File: code/kelin/labs/lab05_pick6.py [hunk 2]
```diff
@@ -1,57 +1,67 @@
...
+        if ticket[index] == winning[index]:
+            matches += 1
+    return matches 
+
+earnings = [] # To add the amount of winnings
+balance = [] # To show the amount of winnings
```
Comment (violette-git) [inline code/kelin/labs/lab05_pick6.py:39]: Balance should be given a value of 0 per lab instructions. Consider your balance like the pile of money you are paying tickets with. Except in this case that pile starts at 0.

**Review by violette-git — CHANGES_REQUESTED** [review]
Consider breaking the lab up into the steps given in the lab instructions. Let me know  if you have any questions.

**Review by brucestull — APPROVED** [review]
[Pick6](https://github.com/PdxCodeGuild/class_HB2/blob/main/1%20Python/labs/05%20Pick6.md)

**Completed**

Great use of variables. Good use of docstrings.

**Pull request merged**